lib/better_errors/editor.rb

```diff
@@ -80,7 +81,7 @@ def url(raw_path, line)
       else
         file = raw_path
       end
-        
+
```

Comment:
Remove this line change?

Reply:
Apologies, it was an automated change. Would you like me to remove it?